app/src/main/java/org/fossasia/badgemagic/adapter/SaveAdapter.kt

```diff
@@ -13,9 +13,11 @@ import org.fossasia.badgemagic.R
 import org.fossasia.badgemagic.data.fragments.BadgeConfig
 import org.fossasia.badgemagic.data.fragments.ConfigInfo
 import org.fossasia.badgemagic.util.MoshiUtils
+import org.fossasia.badgemagic.util.Resource
 
 class SaveAdapter(private val context: Context?, private val list: List<ConfigInfo>, private val listener: OnSavedItemSelected) : RecyclerView.Adapter<SaveAdapter.SaveItemHolder>() {
     private var selectedPosition: Int = -1
+    private var resourceHelper = Resource()
```

Comment:
Where is this variable reassigned?

Reply:
Reasaigned? It is declared for the very first time in this class

Reply:
Exactly. So why is it var?

Reply:
I can use val, I just copied the line above it and edited without noticing. You're a saviour, thanks